documentation/how-to/prepare-for-production.md

```diff
@@ -0,0 +1,193 @@
+
+# Prepare for Production
```

Comment:
What do we mean here? Do we expect users to use the charm not in production? Trying to understanding if this the right title